pkgs/applications/networking/mailreaders/evolution/evolution-ews/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchurl, gnome3, cmake, gettext, intltool, pkg-config, evolution-data-server
+, sqlite, gtk3, webkitgtk, libgdata }:
+
+stdenv.mkDerivation rec {
+  pname = "evolution-ews";
+  version = "3.36.4";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/${pname}/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "0zfq02h3r1qbxak04i49564q4s2ykvkgcyc3krjgndan9lq3kvvn";
+  };
+
+  nativeBuildInputs = [ cmake gettext intltool pkg-config ];
+
+  buildInputs = [
+    evolution-data-server gnome3.evolution
+    sqlite libgdata
+    gtk3 webkitgtk
+  ];
+
+  cmakeFlags = [
+    "-DWITH_MSPACK=OFF"
```

Comment:
Could you add a comment why this is disabled?

Reply:
actually it is better to have mspack on according to https://wiki.gnome.org/Apps/Evolution/Building#Build_evolution-ews
I had it off to get it to work in the first place